src/main/resources/application.properties

```
@@ -24,6 +24,7 @@ management.endpoints.web.exposure.include=mappings
 
 app.admin.emails=${ADMIN_EMAILS:${env.ADMIN_EMAILS:phtcon@ucsb.edu}}
 app.sourceRepo=${SOURCE_REPO:${env.SOURCE_REPO:https://github.com/ucsb-cs156/proj-frontiers}}
+app.webhook.secret=${WEBHOOK_SECRET:${env.WEBHOOK_SECRET:webhook_secret_unset}}
```

```suggestion
app.webhook.secret=${WEBHOOK_SECRET:${env.WEBHOOK_SECRET:unset}}
```

Should we not have a default that is fewer than 10 characters, so that it won't start up with out a valid value?